File: java/core/src/main/java/com/github/fernthedev/lightchat/core/packets/latency/PingPacket.java
package com.github.fernthedev.lightchat.core.packets.latency;

import com.github.fernthedev.lightchat.core.packets.Packet;
import com.github.fernthedev.lightchat.core.packets.PacketInfo;

@PacketInfo(name = "PING_PACKET")
public class PingPacket extends Packet implements LatencyPacket {

}

File: java/core/src/main/java/com/github/fernthedev/lightchat/core/packets/latency/PingPacket.kt
package com.github.fernthedev.lightchat.core.packets.latency

import com.github.fernthedev.lightchat.core.packets.PacketInfo
import com.github.fernthedev.lightchat.core.packets.PacketJSON
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
@PacketInfo(name = "PING_PACKET")
class PingPacket : PacketJSON(), LatencyPacket